phase1/gce/configure-vm.sh

```diff
@@ -21,6 +21,7 @@ KUBEADM_ADVERTISE_ADDRESSES=$(get_metadata "k8s-kubeadm-advertise-addresses")
 KUBEADM_CNI_PLUGIN=$(get_metadata "k8s-kubeadm-cni-plugin")
 KUBEADM_MASTER_IP=$(get_metadata "k8s-kubeadm-master-ip")
 KUBEPROXY_MODE=$(get_metadata "k8s-kubeproxy-mode")
+KUBEADM_FEATURE_GATES=$(get_metadata "k8s-kubeadm-feature-gates")
```

Comment:
we need to define this variable `KUBEADM_FEATURE_GATES` in openstack too.

Reply:
obviously yes ... will add.